src/rtl/gtwvt/gtwvt.c

```diff
@@ -2215,12 +2217,29 @@ static void hb_gt_wvt_MouseEvent( PHB_GTWVT pWVT, UINT message, WPARAM wParam, L
          break;
 
       case WM_RBUTTONUP:
+         if( pWVT->bQuickEdit )
+         {
+            HB_GT_INFO gtInfo;
+
+            gtInfo.pNewVal  = NULL;
+            gtInfo.pNewVal2 = NULL;
+            gtInfo.pResult  = NULL;
+
+            if( HB_GTSELF_INFO( pWVT->pGT, HB_GTI_CLIPBOARDDATA, &gtInfo ) )
+               HB_GTSELF_INKEYSETTEXT( pWVT->pGT, hb_itemGetCPtr( gtInfo.pResult ),
+                                       hb_itemGetCLen( gtInfo.pResult ) );
```

Comment:
It'd simplify the code if `HB_GTI_CLIPBOARDPASTE` would be called here. This option puts the clipboard to the keyboard buffer with a single call.